src/api/mod.rs

```diff
@@ -1923,7 +1923,7 @@ impl ApiResponse {
     }
 
     /// Like convert but produces resource not found errors.
-    pub fn convert_rnf<T: DeserializeOwned>(self, res_err: ApiErrorKind) -> ApiResult<T> {
+    fn convert_rnf<T: DeserializeOwned>(self, res_err: ApiErrorKind) -> ApiResult<T> {
```

Comment:
Privatizing this function allows us to restrict `ApiErrorKind` to `pub(in crate::api)`